test/test-traces.js

```diff
@@ -0,0 +1,482 @@
+import config from 'config';
+import db from '../app/services/db';
+import { expect } from 'chai';
+import { createMockUser, Client, createMockTrace } from './utils';
+import validTraceJson from './fixtures/valid-trace.json';
+import cloneDeep from 'lodash.clonedeep';
+import orderBy from 'lodash.orderby';
+import traces from '../app/models/traces';
+
+const paginationLimit = config.get('pagination.limit');
+/* global apiUrl */
```

Comment:
@vgeorge am just curious how this works / where does this `apiUrl` variable get defined?

Reply:
It is defined at `test--index.js`, which is the entry file for tests:

https://github.com/developmentseed/observe-api/blob/4ea72ccacf936ff580b2b5838f2e7dd1e0c11f76/test/test--index.js#L9-L10